addons/account/migrations/9.0.1.1/tests/test_account.py

```diff
@@ -1,13 +0,0 @@
-# coding: utf-8
```

Comment:
I tried to update the state of the account_full_reconcile module to 'to upgrade' and latest_version to '8.0.1.0'  in the post_migration of account, but migrations are not run. Seems that somehow the latest_version is reset and therefore migration ignored...